src/main/java/stream/flarebot/flarebot/commands/general/PollCommand.java

```diff
@@ -1,19 +1,353 @@
+            if (guild.getPolls().size() == 0 || closedSize > 0) {
+                channel.sendMessage(MessageUtils.getEmbed(sender)
+                        .setDescription("This guild has no polls currently running!" +
+                                (closedSize > 0 && FlareBot.getInstance().getPermissions(channel).hasPermission(member, "flarebot.poll.open")
+                                        ? String.format("\nThis guild has %d closed " + (guild.getPolls().size() == 1 ? "poll" : "polls"), closedSize) : ""))
+                        .setColor(Color.CYAN)
+                        .build()).queue();
+            } else {
+                channel.sendMessage(MessageUtils.getEmbed(sender)
+                        .setDescription(String.format("This guild has %d " + (guild.getPolls().size() == 1 ? "poll" : "polls"), guild.getPolls().size()))
+                        .setColor(Color.CYAN)
+                        .build()).queue();
+            }
+            return;
+        } else {
+            if (args[0].equalsIgnoreCase("create")) {
+                if (args.length >= 2) {
+                    if (guild.getPolls().size() >= 12) {
+                        MessageUtils.sendErrorMessage("You cannot have more that 12 polls at any one time!", channel);
+                        return;
+                    }
+                    String question = MessageUtils.getMessage(args, 1);
+                    if (question.length() > 1024) {
+                        MessageUtils.sendErrorMessage("The question you entered is too long! It has to be 1024 characters or shorter!", channel);
+                        return;
+                    }
+                    Poll poll = new Poll(question, channel);
+                    guild.getPolls().add(poll);
+                    channel.sendMessage(new EmbedBuilder().setTitle("Poll created", null)
+                            .setColor(Color.green).setAuthor(sender.getName(), null, sender.getEffectiveAvatarUrl())
+                            .setDescription(String.format("Poll with ID %d has been created! Remember to set the close-time and the options for this poll!\n" +
+                                    "To open the poll for people to vote on do `poll open`!", guild.getPolls().indexOf(poll) + 1))
+                            .build()
+                    ).queue();
+                    return;
+                }
+            } else if (args[0].equalsIgnoreCase("list")) {
+                EmbedBuilder builder = MessageUtils.getEmbed(sender);
+                if (guild.getPolls().size() == 0) {
+                    channel.sendMessage(MessageUtils.getEmbed(sender)
+                            .setDescription("This guild has no polls currently running!")
+                            .setColor(Color.CYAN)
+                            .build()).queue();
+                    return;
+                }
+                guild.getPolls().stream().limit(12).forEach(poll -> {
```

Comment:
You already limit this, all this would do is make it so we manually override it would not show them. Remove the limit and save time by using a proper for rather than J8's forEach